### ISSUE TITLE
Package update: chromium

### ISSUE THREAD

**Issue by mafigit** [voidlinux/void-packages#7137]
Update to version 59.0.3071.115

**Comment by ebfe** [comment]
Please rebase on top of the actual update commit :)

**Comment by mafigit** [comment]
I think there is a misunderstanding :) This is actually a request/issue to update to a newer chromium version. After posting this, I tried to update the the package myself. But there is a problem with libxml. I get the following error during compilation:
`/libxml_utils.cc:28:29: error: 'XML_PARSE_NOXXE' was not declared in this scope
                             XML_PARSE_NOXXE;     // no external entities`

I found a similar problem in the arch repos: [chromium-vaapi](https://aur.archlinux.org/packages/chromium-vaapi/)
